app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -1023,7 +1023,7 @@ def rbac_user_validate?
     if @edit[:new][:group].blank?
       add_flash(_("A User must be assigned to a Group"), :error)
       valid = false
-    elsif Rbac.filtered([MiqGroup.find_by_id(@edit[:new][:group])].compact).empty?
+    elsif Rbac.filtered(MiqGroup.find(@edit[:new][:group].split(',').delete_if(&:blank?))).empty?
```

Comment:
We need *all* the groups to be OK, not just one of them...

So.. I guess something like 

```
ids = @edit[:new][:group].split(',').delete_if(&:blank?)
...
elsif Rbac.filtered(MiqGroup.find(ids))).count != ids.count`
...